### ISSUE TITLE
Simple Feed Reader

### ISSUE THREAD

**Issue by Breizhux** [SimpleMobileTools/General-Discussion#221]
Hello,

**I'd like to suggest you add a very useful application to your bundle: a simple RSS reader that works!**

I have tried many RSS feed applications on Fdroid, however, none of them satisfy me completely... They either require an account on a server, are poorly designed, or have recurring bugs.

The idea would be an application that doesn't require you to install (or find) a server to accept RSS feeds. I think that many people do not read the news on both their smartphone and their computer. So there is no real need to synchronise the two via a server. And so a simple, single application would fit the bill perfectly.

Secondly, in terms of design, the **[SpaRSS](https://f-droid.org/fr/packages/org.decsync.sparss.floss/)** application from Fdroid **is very simple and does the job** : a **menu grouping the different sites followed**; the **possibility to add/remove sources** by going to the settings; a **favourite folder** that allows articles to be kept aside; a **search tool** to come back to an article later, after reflection.
However, this application has its limits: I've been using it for years, and **it has bugs** that come and go with each version (I'm thinking in particular of a poor management of the background synchronisation of articles)... Also, it **doesn't have a tool to automatically find the available RSS feeds URL** of a blog. It is necessary to find it by hand, which is not an easy task for some blogs. This lack of simplicity could be a barrier for some users...

So I looked for a solution that solves both of these problems, and I found **[Feeder](https://f-droid.org/fr/packages/com.nononsenseapps.feeder/)**. But again, this solution **is not very well designed** in my opinion and does not make the basic functionality very convenient to use. And makes SpaRSS preferable in several ways:
 - On SpaRSS it is possible to "mark as read" an article by dragging it to the left, and "bookmark" it by dragging it to the right. On Feeder, either side will be marked as read, without any possibility to configure the action. Moreover, I'm not sure which graphic framework Feeder is based on, but on my smartphone it is often necessary to drag the article 2-3 times on the side for it to be marked as read... the sensitivity of the movement is very poorly set at this level, whereas SpaRSS is less pretty, but it works.
 - The management of favourites is very well designed on SpaRSS: articles marked as favourites remain in the main list of articles, as they should be. On the other hand, on Feeder, it is not a matter of bookmarking but of pinning. So, the pinned articles are at the top of the main list of articles. And as I have a lot of pinned articles (about ten), when I update the list of articles, I don't immediately see the new articles, but I have to scroll to see them appear.
 - The management of sources on SpaRSS is simple and allows you to add sources in 2-3 clicks, to arrange them by group. On Feeder, the management is much more complex and does not seem to allow to manage groups of sources easily. The only advantage Feeder has over SpaRSS here is that it is able to automatically detect RSS feeds from a blog, making it very easy to find the specific RSS link. But otherwise, it is not very good.
 - Feeder does not have a search tool... And that's a shame...

**In short, what would serve many people well would be a simple RSS reader with well-designed basic functionality, and properly designed source management. A sort of SpaRSS/Feeder that only takes the best of both to get a simple and elegant application!**

Thanks in advance! And most of all, thank you for all the applications you develop that are designed to serve you in the most efficient way possible! I really appreciate all your applications!

I wish you all the best for the future!

**Comment by tibbi** [comment]
hi, that cannot be added as it doesnt fit our suite and requires internet access. Thanks though
